### PR TITLE
images

### PR DESCRIPTION
**Pull request by spaceker19**

### CODE REVIEW

**Review by AmaaniGoose — CHANGES_REQUESTED** [review]
Resolve merge conflicts

File: index.html [hunk 1]
```diff
@@ -102,7 +102,8 @@
 
 		<!-- banner start --> 
 		<div id="banner" class="banner">
-			<div class="banner-image"></div>
```
Comment (AmaaniGoose) [inline index.html:105]: To change background image you have to change the css file in styles.css. Change the image path in the banner-image class to change it

File: index.html [hunk 2]
```diff
@@ -102,7 +102,8 @@
...
+			<div class="https://github.com/spaceker19/animeclubsite/blob/main/images/681016.jpg"></div>
```
Comment (AmaaniGoose) [inline index.html]: Please remove this line

File: index.html [hunk 3]
```diff
@@ -102,7 +102,8 @@
...
+			<div style="background-image: url('https://github.com/spaceker19/animeclubsite/blob/main/images/681016.jpg');"></div>
```
Comment (AmaaniGoose) [inline index.html]: Please remove this line and change background using styles.css only

File: index.html [hunk 4]
```diff
@@ -791,8 +792,17 @@ <h3>Ongoing</h3>
...
+				 <li class="list-group-item">Kaguya Sama Love is War</li>
+				 <li class="list-group-item">One Piece</li>
+				 <li class="list-group-item">Jujitsu Kaisen</li>
+				 <li class="list-group-item">One Punch Man</li>
+				 <li class="list-group-item">ChainsawMan</li>
+				 <li class="list-group-item">Haikyuu</li>
 				   
-				   </div>
+				</div>
 				   <ul class="list-group">
```
Comment (AmaaniGoose) [inline index.html]: Please delete empty lists

File: index.html [hunk 5]
```diff
@@ -813,6 +823,106 @@ <h3>Classics</h3>
 			   </div>
 		   </section>
 		   <!-- section end -->
+		   <!-- section start -->
```
Comment (AmaaniGoose) [inline index.html]: Suggested that these online resources be added into carousel itself. Do you want me to finish it?

**Review by AmaaniGoose — CHANGES_REQUESTED** [review]
Please take a look

File: index.html (hunk 5 above)
Comment (spaceker19) [reply index.html]: yeah please

**Comment by AmaaniGoose** [comment]
Integrated the files

**Pull request closed without merging**